src/audio-worklet/free-queue/src/interface/C/README.md

```diff
@@ -0,0 +1,59 @@
+# FreeQueue C Interface
+
+A single header C API implementation to access FreeQueue functionality from C/C++ programs.
```

Comment:
It's better to talk about the thread safety upfront. Can you add some notes here?

Reply:
Sure.